client/src/shared/layouts/LayoutLearning.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
-import NextIcon from '../icons/next';
-import PreviousIcon from '../icons/previous';
-import Sidebar from './sidebar';
+import NextIcon from '../icons/NextIcon';
+import PreviousIcon from '../icons/PreviousIcon';
+import Sidebar from './Sidebar';
```

Comment:
Always use module path aliases
```suggestion
import NextIcon from '@/src/shared/icons/NextIcon';
import PreviousIcon from '@/src/shared/icons/PreviousIcon';
import Sidebar from '@/src/shared/layouts/Sidebar';
```